OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -271,7 +280,15 @@ void LoadWaypoints(IniSection waypointSection)
 						new OwnerInit("Neutral")
 					};
 
-					Map.ActorDefinitions.Add(new MiniYamlNode("waypoint" + kv.First, ar.Save()));
+					var waypointName = "waypoint" + kv.First;
+					if (kv.First == 25)
```

Comment:
What's the source of these numbers?  Are you sure they are the same for both TD and RA?